docs/docs/exclusion.md

```diff
@@ -0,0 +1,12 @@
+## Excluding registries, repositories, and images
```

Comment:
do we need to add this to sidebar?

Reply:
im not sure how to test that, is it autogenerated?

Reply:
ah looks like it's autogenerated https://github.com/Azure/eraser/blob/main/docs/sidebars.js#L17

but we need to define the position
https://github.com/Azure/eraser/blob/main/docs/docs/installation.md?plain=1#L3

can we create a parent item to this? perhaps "how to use eraser", "configuration" or something?